components/serializer.rst

```diff
@@ -1486,11 +1486,11 @@ having unique identifiers::
 
     $encoder = new JsonEncoder();
     $defaultContext = [
-        AbstractNormalizer::CIRCULAR_REFERENCE_HANDLER => function (object $object, string $format, array $context): string {
+        AbstractNormalizer::CIRCULAR_REFERENCE_HANDLER => function (object $object, ?string $format, array $context): string {
             return $object->getName();
         },
     ];
-    $normalizer = new ObjectNormalizer(null, null, null, null, null, null, $defaultContext);
+    $normalizer = new ObjectNormalizer(defaultContext: $defaultContext);
```

Comment:
This should be reverted IMO as Symfony's BC promise does not cover named arguments.

Reply:
ok I'll note for next time, So all doc must be without named arguments, or just old are not changed ?

Reply:
the whole documentation except for attributes